Include/cpython/code.h

```diff
@@ -3,6 +3,8 @@
 #endif
 
 typedef uint16_t _Py_CODEUNIT;
+// Each oparg must fit in the second half of _Py_CODEUNIT, hence 8 bits.
+#define _Py_MAX_OPARG 255
```

Comment:
We already have UINT8_MAX, I don't think we need another const for 255 :slightly_smiling_face:

Reply:
Yeah, I meant to get rid of this one.

Reply:
FYI, I'm still going to fix this. 🙂